db/db_impl/db_impl_write.cc

```diff
@@ -215,6 +215,7 @@ Status DBImpl::MultiBatchWriteImpl(const WriteOptions& write_options,
     // and it would not notify the threads in this WriteGroup. So we must make someone in
     // this WriteGroup to complete it and leader thread is easy to be decided.
     if (is_leader_thread) {
+      assert(write_thread_.CompleteParallelMemTableWriter(&writer));
```

Comment:
From what I remember `assert` will be optimize out in release build. Do something like the following:
```
bool ok __attribute__((__unused__)) write_thread_.CompleteParallelMemTableWriter(&writer);
assert(ok);
```

Reply:
I'm confused. Without calling `CompleteParallelMemTableWriter` before, how do the leader make sure all other parallel writers in the group has finished at this point?

Reply:
Because leader_thread will be blocked in the loop until all followers have finished their tasks.

Reply:
So in release build, we do not need to run `CompleteParallelMemTableWriter`.

Reply:
I prefer calling `CompleteParallelMemTableWriter` anyway, that's the contract with `WriteThread`.

Reply:
Ok.